From: Norbert Preining <norbert@preining.info>
Date: Fri, 21 Jul 2017 10:18:34 +0900
Subject: Support new date format of LaTeX kernel in TL2017

LaTeX has changed its date format in TL2017, so AucTeX needs to recognize
this format, too, otherwise warnings are issued.

Patch by Markus Kohm on the TeX Live mailing list.
---
 tex-buf.el |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- auctex.git.orig/tex-buf.el
+++ auctex.git/tex-buf.el
@@ -1582,7 +1582,7 @@
 	 (setq TeX-command-next TeX-command-default))
 	((re-search-forward
 	  "^\\(\\*\\* \\)?J?I?p?\\(La\\|Sli\\)TeX\\(2e\\)? \
-\\(Version\\|ver\\.\\|<[0-9/]*\\(?:u[^>]*\\)?>\\)" nil t)
+\\(Version\\|ver\\.\\|<[0-9/-]*\\(?:u[^>]*\\)?>\\)" nil t)
 	 (let* ((warnings (and TeX-debug-warnings
 			       (TeX-LaTeX-sentinel-has-warnings)))
 		(bad-boxes (and TeX-debug-bad-boxes
